com.github.ztefn.haguichi.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.github.ztefn.haguichi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "haguichi",
+    "finish-args": [
+        "--filesystem=/var/lib/logmein-hamachi",
```

Comment:
Does it depends on having hamachi being installed on host? Can't it be bundled with this package?

Reply:
No, it's just too check if hamachi configuration is present. If hamachi is not installed or properly configured Haguichi will tell the user this. Hamachi itself is proprietary software, so I would not recommend bundling.

Reply:
> No, it's just too check if hamachi configuration is present. If hamachi is not installed or properly configured Haguichi will tell the user this. 

In such case, please add `:ro` at the end of the `--filesystem` permission.


> Hamachi itself is proprietary software, so I would not recommend bundling.

You can still bundle it using extra-data which will download it and build/install it on the user's machine.

Reply:
> In such case, please add `:ro` at the end of the `--filesystem` permission.

Done!

> You can still bundle it using extra-data which will download and build/install it on the user's machine.

Even if hamachi could technically run within Flatpak context (which I highly doubt), I don't want some user to mix up Haguichi and Hamachi even more then they already do and blame Haguichi when their connection fails and/or [come to me for support](https://github.com/ztefn/haguichi/issues/22)...

Reply:
Can you add a note to appdata file that Hamachi is required to be installed on host for Haguichi to work?

Reply:
@barthalion The summary in the appdata should be a big giveaway already:
`<summary>A graphical frontend for Hamachi</summary>`

I suppose an additional note could be added to the `<description>`. But I don't plan too make another release anytime soon and the appdata file is contained in the release package.

Reply:
Sure, we can backport the patch here until it’s part of stable release.

Reply:
@barthalion Here is the patch: https://github.com/ztefn/haguichi/commit/08700dfe944c99bb40be4ad9b9e0e0a6c3d92c99